src/Cron_Command.php

```diff
@@ -38,18 +38,18 @@ public function __construct() {
 	 *
 	 * We also have helper to easily specify scheduling format:
 	 *
-	 *  Entry                  | Description                                | Equivalent To
-	 *  -----                  | -----------                                | -------------
-	 *  @yearly (or @annually) | Run once a year, midnight, Jan. 1st        | 0 0 1 1 *
-	 *  @monthly               | Run once a month, midnight, first of month | 0 0 1 * *
-	 *  @weekly                | Run once a week, midnight between Sat/Sun  | 0 0 * * 0
-	 *  @daily (or @midnight)  | Run once a day, midnight                   | 0 0 * * *
-	 *  @hourly                | Run once an hour, beginning of hour        | 0 * * * *
+	 * | Entry                  | Description                                | Equivalent To
+	 * | -----                  | -----------                                | -------------
+	 * | @yearly (or @annually) | Run once a year, midnight, Jan. 1st        | 0 0 1 1 *
+	 * | @monthly               | Run once a month, midnight, first of month | 0 0 1 * *
+	 * | @weekly                | Run once a week, midnight between Sat/Sun  | 0 0 * * 0
+	 * | @daily (or @midnight)  | Run once a day, midnight                   | 0 0 * * *
+	 * | @hourly                | Run once an hour, beginning of hour        | 0 * * * *
 	 *
 	 * You may also schedule a job to execute at fixed intervals, starting at the time it's added or cron is run.
 	 * This is supported by following format:
 	 *
-	 * @every <duration>
+	 *  @every <duration>
```

Comment:
Change space to some other character so that it does not get corrected on phpcbf run accidentaly.